airbyte-integrations/connectors/source-google-sheets/source_google_sheets/spec.yaml

```diff
@@ -73,14 +73,46 @@ connectionSpecification:
               airbyte_secret: true
               examples:
                 - '{ "type": "service_account", "project_id": YOUR_PROJECT_ID, "private_key_id": YOUR_PRIVATE_KEY, ... }'
-authSpecification:
-  auth_type: oauth2.0
-  oauth2Specification:
-    rootObject:
-      - credentials
-      - 0
-    oauthFlowInitParameters:
-      - ["client_id"]
-      - ["client_secret"]
-    oauthFlowOutputParameters:
-      - ["refresh_token"]
+advanced_auth:
+  auth_flow_type: oauth2.0
+  predicate_key:
+    - credentials
+    - auth_type
+  predicate_value: Client
+  oauth_config_specification:
+    complete_oauth_output_specification:
+      type: object
+      additionalProperties: false
+      properties:
+        refresh_token:
+          type: string
+          path_in_connector_config:
+            - credentials
+            - refresh_token
+        access_token:
```

Comment:
I think we can skip it as it is not present in the spec

Reply:
Fixed